.gitlab-ci.yml

```diff
@@ -307,6 +307,20 @@ build:edge+flambda:
 build:base:dev:
   extends: .build-template:base:dev
 
+# Build using native dune rules
+build:base:dev:dune:
+  stage: build
+  image: $EDGE_IMAGE
+  interruptible: true
+  extends: .auto-use-tags
+  script:
+    - cp theories/dune.disabled theories/dune
+    - cp user-contrib/Ltac2/dune.disabled user-contrib/Ltac2/dune
+    - dune build coq-core.install coq-stdlib.install coq.install
+    - ls _build/install/default/lib/coq/theories/Reals/Reals.vo
+    - ls _build/install/default/lib/coq/user-contrib/Ltac2/Ltac2.vo
+  only: *full-ci
```

Comment:
while testing the new job you can put it in light CI, that way we don't need to run everything.